colossus-metrics/src/main/scala/colossus/metrics/logging/ColossusLogging.scala

```diff
@@ -0,0 +1,46 @@
+package colossus.metrics.logging
+
+import org.slf4j.{Logger, LoggerFactory}
+
+trait ColossusLogging {
```

Comment:
Not sure we need to call it `ColossusLogging`, could just do `Logging`, but I'm ok either way.

Reply:
FWIW: Not sure if anyone saw my previous comment, but there is already a thing that does this: 
https://github.com/typesafehub/scala-logging

Reply:
This is pretty simple, we don't really need to pull in another dependency.

Reply:
Also scala-logging does the magic using macros, which might cause issues with debugging if I read the bottom of the  scala-logging readme correctly.

Reply:
What kind of issues, do you think that will cause? It has a few other things in it too, such as lazy/non lazy implementations which is nice if you don't want to instantiate loggers which don't get used.

NBD either way, I figured we would avoid reinventing the wheel :)

Reply:
They have a section [Debugging Scala in Intellij](https://github.com/typesafehub/scala-logging#debugging-scala-in-intellij). I haven't tried it myself, so I don't know the details.

Luckily this wheel is pretty small, plus didn't you write this code? Can't get a better implementation than that!

Reply:
The code did look vaguely familiar, but I wasn't sure if it was me :-)

In either event it's not big deal as like you are saying, it's a small wheel.  But I felt like I should at least point out that the wheel does exist.